### PR TITLE
populate sessions api with all fields

### LINKED ISSUE #3698: Populate sessions api with corresponding fields
Additionally Implement speakers and session_types api only with the fields that are strictly required (Please do not add any other fields, it will be done in other issue only, PR may be rejected if you do otherwise. We are strictly trying to stick to one PR per issue.)

### PR DESCRIPTION
**Pull request by shubham-padia**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
populate sessions api with all fields

#### Changes proposed in this pull request:

- rename submission_date to submitted_at to follw db naming schema
-
-

Fixes #3698

### CODE REVIEW

**Comment by magdalenesuo** [comment]
@shubham-padia Do we need stand alone \sessions?

**Comment by shubham-padia** [comment]
@magdalenesuo I don't think so, making changes

**Comment by shubham-padia** [comment]
@magdalenesuo changes made

**Comment by shubham-padia** [comment]
@SaptakS that falls under #3712 or some new issue. It is out of scope of this PR. Please open a new issue if it doesn't fall under #3712 :smile:

**Comment by shubham-padia** [comment]
@magdalenesuo I don't understand, can you rephrase.

**Comment by magdalenesuo** [comment]
@shubham-padia please include '/microlocations/<int:microlocation_id>/sessions' in session list

**Comment by shubham-padia** [comment]
@magdalenesuo  I'll add if you and @SaptakS want this change in this PR. But I don't think finding a bug in another related api and asking for changes here is the right way to go about PRs. PRs should address the said issue. You could have just opened a new issue and assigned it to me.

**Comment by magdalenesuo** [comment]
@shubham-padia  it is not

>  finding a bug in another related api

since we were completing session API field and relationships in this PR, we only wanted the left out api relation with session to be completed here itself.
Since microlocation issue concentrated only on the relationship between microlocation - session, session - microlocation url endpoints were not looked into. 
Removing from detail could have be done in the refactoring issue.

**Comment by shubham-padia** [comment]
@magdalenesuo @SaptakS Moved the`/microlocations/<int:microlocation_id>/sessions` from session_detail to session_list along with required modifications. please review again

**Comment by niranjan94** [comment]
> that falls under #3712 or some new issue. It is out of scope of this PR. Please open a new issue if it doesn't fall under #3712 

@shubham-padia 

There maybe a separate issue open for it. But that is to deal with the code that has already been merged. Any code that is about to be merged could and ideally should fix since it's still yet to be merged. 

A small analogy ... Let's say we have an issue open to fix all spelling mistakes in a project. That doesn't mean any new PR being opened can have spelling mistakes and be later resolved in that previous issue. It doesn't make sense does it ? :)

**Comment by shubham-padia** [comment]
@niranjan94 

> That doesn't mean any new PR being opened can have spelling mistakes 

- There is no new issue that is introduced in this PR which has to addressed in #3712 down the road.
- The issue mentioned in not related to /sessions api endpoint 
- This change is related to microlocations api endpoint for which code was merged earlier.

Nevertheless, let's not elongate this thread, the relevant changes have been made :smile:

**Comment by niranjan94** [comment]
Correct @shubham-padia ... I can see that this PR is completely resolved, approved and ready-to-merge. 

But since I saw a comment regarding scopes, Just wanted to make it clear what scopes for issues and PRs should be in this case so that any one else looking at this would know and take care of it in future ... 

> Nevertheless, let's not elongate this thread

Occasionally, it's alright to elongate threads if there is something worthwhile going on in it 😉

File: app/api/sessions.py [hunk 1]
```diff
@@ -31,6 +31,17 @@ class Meta:
     comments = fields.Str()
     starts_at = fields.DateTime(required=True)
     ends_at = fields.DateTime(required=True)
+    language = fields.Str()
+    level = fields.Str()
```
Comment (niranjan94) [inline app/api/sessions.py]: Level field is duplicate

Comment (shubham-padia) [inline app/api/sessions.py]: Yeah, sorry noticed that today, removed in today's PR

Comment (niranjan94) [inline app/api/sessions.py]: Cool. I'm only now checking all the PRs and code I missed ...
